rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/charts/MemoryLineChart.java

```diff
@@ -73,22 +70,20 @@ public void processResult(String result) {
         if (array != null) {
```

Comment:
The next code looks very the same as in MBeanTimeAreaChart.java 
Could you externalize it to avoid duplication?

Reply:
Yeah. it is the same code. I will remove one of it